fetch/fetch.go

```diff
@@ -361,7 +360,10 @@ func (f *Fetch) FetchRequestHandler(ctx context.Context, data []byte) []byte {
 		}
 		res, err := db.Get(r.Hash.Bytes())
 		if err != nil {
-			f.log.WithContext(ctx).Warning("remote peer requested non existing hash %v %v", r.Hash.Hex(), err)
+			f.log.WithContext(ctx).With().Warning("remote peer requested non existing hash",
```

Comment:
move this to info or even debug

Reply:
changed to info for now. i''d like to verify that this is fixed first. if it proved to be too verbose in the future, i'll move it down to debug if it's deem normal.

Reply:
why it should be info? this seems useful only for developer. tells nothing to operator about health of this node, or about the progress

Reply:
tried to run a local node today and can confirm that this log is pretty much useless to me

Reply:
i agree. and you can say that about a lot of INFO log in the codebase.
but what i would like to do is to verify that this PR does fix the issue. and the only way to find out is to see if this is still happening post-merge and why. unless you have a way to selectively turn on dbg log for a module.

Reply:
but you can turn on debug log only for a single module

Reply:
i misspoke. i meant unless i can turn dbg on `dynamically`. otherwise it will require another change to set it back to non-dbg.